### DISCUSSION TITLE
Amazing tool, thank you!

### DISCUSSION THREAD

**Discussion by Pencilvesterr** [nickcoutsos/keymap-editor#180] [General]
This is such a helpful tool for those creating their first layout! It helps put the ZMK documentation into a much better context and allows easy + quick prototyping. The demo layout was a big help in understanding how to set up home row mods. You've also killed it with UI design and polish, so thank you for all your time and effort in helping out this community!

**Comment by nickcoutsos** [comment]
Many thanks, I'm glad it's working for you!
